opm/output/eclipse/AggregateUDQData.hpp

```diff
@@ -121,13 +124,15 @@ class AggregateUDQData
 
     /// Aggregate 'IUAP' array (ICharArrayNullTermnteger) for all UDQ data  (1 integer pr UDQ constraint used)
     WindowedArray<int> iUAP_;
-#if 0
+
     /// Aggregate 'DUDW' array (Double Precision) for all UDQ data. (Dimension = max no wells * noOfUDQ's)
     WindowedArray<double> dUDW_;
-
+    
+    /// Aggregate 'DUDG' array (Double Precision) for all UDQ data. (Dimension = max no wells * noOfUDQ's)
```

Comment:
I guess this should read: (Dimension = max no *groups* * noOfUDQ's)

Reply:
Agree it is wrong, actually the dimension should be (max no groups + 1) * noOfUDQ'a) - will be corrected